### PR TITLE
setup project

### PR DESCRIPTION
**Pull request by PedroH-Neves**
Here i setup the react app using create react app method so then i create a single page app and I setup the linters.

### CODE REVIEW

**Review by amon-cofie — CHANGES_REQUESTED** [review]
Hi @PedroH-Neves , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional readme file and does not have a descriptive pull request summary. Please add these and submit for another review 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (arslanbisharat) [inline .babelrc:1]: - [ ] Thank you for updating your readme and pull request but your pull request is still not enough descriptive for it to get approval. I request you to kindly update it and add information 'what this PR is about?'.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @PedroH-Neves, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @PedroH-Neves, 

To highlight 👍🏽 
✔️ README is descriptive
✔️ App runs smoothly

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**